locales/ml-IN.ftl

```diff
@@ -0,0 +1,21 @@
+title = ഫയര്‍ഫോക്സ് ക്വാണ്ടം റിലീസ് പ്രചാരണപ്രവര്‍ത്തനങ്ങള്‍
+project-name = ഫയര്‍ഫോക്സ് ക്വാണ്ടം റിലീസ്
+main-title = =ഫയര്‍ഫോക്സ് തിരിച്ചടിക്കുന്നു!
+main-description = പുതിയ ഫയര്‍ഫോക്സിനെപ്പറ്റിയുള്ള വിവരങ്ങള്‍ പ്രചരിപ്പിക്കുവാന്‍ ഞങ്ങളെ സഹായിക്കുക! 14-ാം തീയ്യതി ചൊവ്വാഴ്ച 14:00 യുടിസി സമയത്ത് പുതിയ ഫയര്‍ഫോക്സില്‍ എന്താണ് നിങ്ങള്‍ക്ക് ഇഷ്ടപ്പെട്ടത് എന്ന് #NewFirefox എന്ന ഹാഷ്ടാഗില്‍ ട്വീറ്റ് ചെയ്യുക അല്ലെങ്കില്‍ ഔദ്യോഗിക ഫയര്‍ഫോക്സിന്റെ ട്വീറ്റുകള്‍ റീട്വീറ്റ് ചെയ്യുക
+thunderclap-cta = ഫയര്‍ഫോക്സ് ക്വാണ്ടത്തിനെ പിന്‍തുണക്കുക!
+thunderclap-link = https://www.thunderclap.it/projects/64595-firefox-quantum-launch-day
+official-tweet-cta = ഔദ്യോഗിക ട്വീറ്റുകള്‍ റീട്വീറ്റ് ചെയ്യുക
+tweet-1 = #NewFirefox 🦊 പുതിയ ഫയര്‍ഫോക്സ് ഇന്ന് പുറത്തിറങ്ങി. ഇത് നേരത്തേയുള്ളതിനേക്കാളും രണ്ടിരട്ടി വേഗതകൂടിയതാണ്. 💥 ഡൌണ്‍ലോഡ് ചെയ്യുക :https://mzl.la/newfirefox
```

Comment:
there seems to be a space missing before the URL. Is that on purpose?

Reply:
corrected that